DIPs/DIP1xxx-JS.md

```diff
@@ -0,0 +1,121 @@
+# The Deprecation Process
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | 10XX                                                            |
+| Review Count:   | 0                                                               |
+| Author:         | Jack Stouffer <jack@jackstouffer.com>                           |
+| Implementation: | N/A                                                             |
+| Status:         |                                                                 |
+
+## Abstract
+
+In order to incrementally improve D or it's standard library, it's often necessary to
+mark features or functions for future removal. This document proposes a standardized
+process for language maintainers to remove public features. This process would be 
+used across DMD, Druntime, and Phobos.
+
+## Contents
+* [Rationale](#rationale)
+* [Description](#description)
+* [Acknowledgements](#acknowledgements)
+* [Reviews](#reviews)
+
+## Rationale
+
+There is general disagreement on the best and/or accepted way to remove public
+features. Each deprecation ends up being handled slightly differently depending on
+who's handling the pull requests. Standardizing the process makes sure that
+deprecations are done very publicly and carefully, so as to minimize breakage and
+to provide clear fixes for user code.
+
+## Description
+
+The following lays out the rules to follow to deprecate a part of D.
+
+A symbol or feature must not be marked for removal on a specific date, but rather on a
```

Comment:
"on a specific release" -> for a specific release

Reply:
Things happen on a date, not for a date.